src/GitUtilities.js

```diff
@@ -20,8 +20,7 @@ export default class GitUtilities {
 
   @logger.logifySync()
   static commit(message) {
-    // Use echo to allow multi\nline strings.
-    ChildProcessUtilities.execSync("git commit -m \"$(echo \"" + message + "\")\"");
+    ChildProcessUtilities.execSync("git commit -m \"" + message + "\"");
```

Comment:
We have tested it on Windows 7 and it doesn't work. The `EOL`s terminate the command on windows and we end up with just a "Publish" message:

![publish](https://cloud.githubusercontent.com/assets/298275/22469512/8fdf549a-e7cc-11e6-85bb-24c6dfd15868.png)

Using `JSON.stringify` doesn't work either (throws an error, see below).

![error_json](https://cloud.githubusercontent.com/assets/298275/22469607/efc40310-e7cc-11e6-85de-dba98af8434c.PNG)